Commands/Commands/utils.js

```diff
@@ -116,15 +116,24 @@ commands.stats = {
           inline: true
         }
       )
-      msg.author.openDM().then((e) => {
-        e.sendMessage('', false, {
+      msg.author.openDM().then((dm) => {
+        dm.sendMessage('', false, {

```

Comment:
Was renaming this really necessary?

Reply:
Since we use `e` for errors, its probably a good idea not to use it elsewhere.